spec/child-apps/uses-loader/uses-loader.spec.js

```diff
@@ -0,0 +1,38 @@
+const activeHash = `#uses-loader`;
+
+export default function() {
```

Comment:
This is a new test that uses the old version of the declareChildApplication api. This test is to ensure there is no regression with the changes in this pr.